cli.js

```diff
@@ -2,19 +2,25 @@
 'use strict';
 const meow = require('meow');
 const opn = require('opn');
+const getStdin = require('get-stdin');
+const tempWrite = require('temp-write');
+const fileType = require('file-type');
 
 const cli = meow(`
 	Usage
 	  $ opn <file|url> [--wait] [-- <app> [args]]
+	  $ stdout | opn [--wait] [--ext] [-- <app> [args]]
 
 	Options
 	  --wait  Wait for the app to exit
+	  --ext   Choose temporary file extension for stdin
```

Comment:
I leave the exact description to @sindresorhus to decide but this should probably be reworded to `File extension for stdin` or something.